lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -489,6 +489,7 @@ def cmd_download(*args)
 
     # Go through each source item and download them
     src_items.each { |src|
+      src = ::File.expand_path(src, client.fs.dir.getwd)
```

Comment:
I think you want the destination path for `download`.

Reply:
destination path worked fine for some reason
```
meterpreter > download ~/file ~/file2 
[*] Downloading: /home/auxilus/file -> ~/file2/file
[*] Downloaded 10.00 B of 10.00 B (100.0%): /home/auxilus/file -> ~/file2/file
[*] download   : /home/auxilus/file -> ~/file2/file
meterpreter > 

```

Reply:
I'm just going to remove this line due to the issue mentioned at https://github.com/rapid7/metasploit-framework/pull/13954#discussion_r518360486 and also because this is using a built in Ruby method to expand a remote path, which won't work as the Ruby interpreter on the local system will have no knowledge about the target file system and therefore won't be able to expand the path properly.